app/views/media_objects/_mejs4_player_js.html.erb

```diff
@@ -9,6 +9,7 @@
 // Define MediaElement 4 configuration here
 var mejs4AvalonPlayer = new MEJSPlayer({
   currentStreamInfo: <%= section_info.to_json.html_safe %>,
+  currentUserExists: <%= current_user_exists %>,
   features: ['playpause', 'current', 'progress', 'duration', 'trackScrubber', 'volume', 'quality', 'createThumbnail', 'addToPlaylist', 'fullscreen'],
```

Comment:
I was thinking that we could just skip the 'addToPlaylist' feature here if the user `cannot? :create, Playlist`.  If that works out then we wouldn't need to pass anything into the JS.

Reply:
Ok, I'll try implementing this.

Reply:
Pushing up this approach now.  I added an extra global JS var to store the features array which I'm not super thrilled about, but seems better than repeating the entire array text in a conditional block.